core/src/main/java/org/elasticsearch/search/aggregations/bucket/DeferringBucketCollector.java

```diff
@@ -54,6 +54,16 @@ public Aggregator wrap(final Aggregator in) {
         return new WrappedAggregator(in);
     }
 
+    /**
+     * Unwrap an aggregator. Used for testing.
+     */
+    public static Aggregator unwrap(Aggregator in) {
```

Comment:
Maybe we could do `searchContext.clearReleasables(Lifetime.PHASE)` like the main code does since all aggregators are supposed to register themselves on to the search context.